torch/csrc/lazy/backend/backend_interface.h

```diff
@@ -119,7 +119,7 @@ class TORCH_API BackendRegistrar {
   BackendRegistrar(const BackendImplInterface* backend_impl_interface);
 };
 
-inline const BackendImplInterface* TORCH_API getBackend() {
+static inline const BackendImplInterface* getBackend() {
```

Comment:
we don't need TORCH_API macro here?

Reply:
For a static inline function in a header file, we don't need it. I removed it according to a lint complain.